Unix/pal/alloc.c

```diff
@@ -119,16 +120,12 @@ void PAL_DumpAllocList()
     if (!_list)
         return;
 
-    printf("\nWARNING: one or more blocks still allocated!\n");
+    trace_DumpAllocList_Warning();    
```

Comment:
What, exactly, does this do?

Reply:
Not sure what you're asking here.  It's printing the same thing, but to the log file.